tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -12,15 +14,69 @@
 from sentry.testutils.helpers.datetime import before_now, iso_format
 
 
-class TestIdentfiyStacktracePaths(TestCase):
+class BaseDeriveCodeMappings(TestCase):
     def setUp(self):
         self.organization = self.create_organization(
             status=OrganizationStatus.ACTIVE,
         )
-        self.project = self.create_project(
+        self.project = self.create_project(organization=self.organization)
+
+
+class TestJavascriptDeriveCodeMappings(BaseDeriveCodeMappings):
+    def setUp(self):
+        super().setUp()
+        self.test_data = {
+            "platform": "javascript",
+            "stacktrace": {
+                "frames": [
+                    {
+                        "module": "@sentry/browser/node_modules/@sentry/core/esm/hub",
+                        "filename": "../node_modules/@sentry/browser/node_modules/@sentry/core/esm/hub.js",
+                        "abs_path": "webpack:///../node_modules/@sentry/browser/node_modules/@sentry/core/esm/hub.js",
+                        "in_app": False,
+                    },
+                    {
+                        "module": "app/utils/handleXhrErrorResponse",
+                        "filename": "./app/utils/handleXhrErrorResponse.tsx",
+                        "abs_path": "webpack:///./app/utils/handleXhrErrorResponse.tsx",
+                        "in_app": True,
+                    },
+                ]
+            },
+        }
+        self.event_data = self.store_event(data=self.test_data, project_id=self.project.id).data
+
+    def test_find_stacktrace_paths_single_project(self):
+        stacktrace_paths = identify_stacktrace_paths(self.event_data)
+        assert stacktrace_paths == ["./app/utils/handleXhrErrorResponse.tsx"]
```

Comment:
This shows that `../node_modules/@sentry/browser/node_modules/@sentry/core/esm/hub.js` will be ignored.